src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -1120,6 +1140,8 @@ impl NamespacedOrchestrator for NamespacedKubernetesOrchestrator {
         let security_context = if let Some(fs_group) = self.config.service_fs_group {
```

Comment:
how does this context relate to `container_security_context`? Can we set everything on the pod context?

Reply:
Some of the fields only exist at the container level like `allow_privilege_escalation` and `se_linux_options`. I think I could have put more of the fields in the PodSecurityContext but I'd prefer to leave it this way.